src/Symfony/Component/Translation/Translator.php

```diff
@@ -37,54 +37,36 @@ class Translator implements TranslatorInterface, TranslatorBagInterface, LocaleA
      */
     protected $catalogues = [];
 
-    /**
-     * @var string
-     */
-    private $locale;
+    private string $locale;
 
     /**
-     * @var array
+     * @var string[]
      */
-    private $fallbackLocales = [];
+    private array $fallbackLocales = [];
 
     /**
      * @var LoaderInterface[]
      */
-    private $loaders = [];
+    private array $loaders = [];
 
-    /**
-     * @var array
-     */
-    private $resources = [];
+    private array $resources = [];
 
-    /**
-     * @var MessageFormatterInterface
-     */
-    private $formatter;
+    private MessageFormatterInterface $formatter;
 
     /**
      * @var string
```

Comment:
why keeping a `string` phpdoc if you make it `?string` ?

Reply:
fixed in 1516fd39114f2f19942a1ba4cf80ec868d46e181